int8 color
---
int16 h_min 
int16 h_max
int16 s_min
int16 s_max
int16 v_min
int16 v_max
